Assets/scripts/incompleto/Destroyer_UI_inicial_background.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Destroyer_UI_Initial_background : MonoBehaviour
+
+{
+    // Start is called before the first frame update
+    void Start()
+    {
+ 
+    }
+
+    public float timeRemaining = 10;
+    public int Get_phase() { return phase; }
```

Comment:
É melhor declarar as variáveis globais no início da classe, acima de todas as funções.

Reply:
Pode excluir a função Get_phase(), creio que não será necessária. Pode usar a chamada do game manager já dentro da função pública principal.